src/Symfony/Component/Messenger/HandleTrait.php

```diff
@@ -13,15 +13,17 @@
 
 use Symfony\Component\Messenger\Exception\LogicException;
 use Symfony\Component\Messenger\Stamp\HandledStamp;
+use Symfony\Component\Messenger\Stamp\SentStamp;
 
 /**
- * Leverages a message bus to expect a single, synchronous message handling and return its result.
+ * Leverages a message bus to expect a single, message handling and return its result.
```

Comment:
```suggestion
 * Leverages a message bus to expect a single, message handling and return its result (for synchronous)
```

wdyt, also perhaps improve the desc of the `handle()` method ?

Reply:
Yeah, I was thinking how to improve the docblock documentation to make it clearer. This trait now can handle async message handling too, so either we don't mention the sync/async terms at all or we mention both. I prefer the simpler one, but it's true that we can't guarantee a single message handling for async, so it's probably better to mention both terms.

I'll update soon, thanks!

Reply:
Updated!